specification/translation/Azure.AI.TextTranslation/models-translate.tsp

```diff
@@ -144,35 +144,37 @@ model TranslationResult {
 union TextType {
   string,
 
-  /** Plain text. */
+  @doc("Plain text.")
   Plain: "Plain",
 
-  /** HTML-encoded text. */
+  @doc("HTML-encoded text.")
   Html: "Html",
 }
 
+#suppress "@azure-tools/typespec-azure-core/use-extensible-enum" "Enums should be defined without the `@fixed` decorator."
+#suppress "@azure-tools/typespec-azure-core/no-enum" "Azure services should not use the enum keyword."
```

Comment:
Can you open an issue for yourself to update these to more meaningful suppression comments? These look like the errors that are reported instead of the reason why your service needs to suppress these warnings. Can be in a separate PR